kolibri/plugins/coach/assets/src/views/plan/LessonResourceSelectionPage/index.vue

```diff
@@ -298,6 +346,21 @@
         addToWorkingResources: 'ADD_TO_WORKING_RESOURCES',
         removeFromSelectedResources: 'REMOVE_FROM_WORKING_RESOURCES',
       }),
+      getBookmarksData() {
+        this.bookmarks.forEach(bookmark => {
+          ContentNodeResource.fetchModel({ id: bookmark.contentnode_id }).then(data => {
+            this.bookmarkNodes.push(data);
+          });
+        });
+      },
```

Comment:
Another place where we can optimize against making a call for every Bookmark's ContentNode